website/content/api-docs/secret/databases/redis.mdx

```diff
@@ -0,0 +1,78 @@
+---
+layout: api
+page_title: Redis - Database - Secrets Engines - HTTP API
+description: >-
+  The Redis plugin for Vault's database secrets engine generates [ACL credentials](https://redis.io/docs/manual/security/acl/)
+  to access Redis servers.
+---
+
+# Redis Database Plugin HTTP API
+
+The Redis database plugin is one of the supported plugins for the database
+secrets engine. This plugin generates database credentials dynamically based on
+configured roles for the Redis database.
+
+## Configure Connection
+
+In addition to the parameters defined by the [Database
+Secrets Engine](/api-docs/secret/databases#configure-connection), this plugin
+has a number of parameters to further configure a connection.
+
+| Method | Path                     |
+| :----- | :----------------------- |
+| `POST` | `/database/config/:name` |
+
+### Parameters
+
+- `plugin_name` `(string: <required>)` – Specifies the name of the plugin to use for this connection.
+
```

Comment:
Seems like other plugins do not define the plugin_name parameter since it is part of the generic database engine params from index.mdx

Reply:
Got it! Thank you so much!